types/prettier/index.d.ts

```diff
@@ -21,7 +21,8 @@ export interface FastPath<T = any> {
 
 export type BuiltInParser = (text: string, options?: any) => AST;
 export type BuiltInParserName =
-    | 'babylon'
+    | 'babylon' // deprecated
+    | 'babel'
```

Comment:
Could you add the `babel-flow` parser as well?

Reply:
Thanks, just did.